.github/workflows/deploy-prod.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
         id: meta
         uses: docker/metadata-action@v3
         with:
-          images: eu.gcr.io/digdir-fdk-infra/dataset-catalog-gui
+          images: eu.gcr.io/digdir-fdk-infra/dataset-catalogue-gui
```

Comment:
Hvorfor navneendringen?

Reply:
Mener det ble gjort en refactor tidligere for å rename alt til amerikansk "catalog".

Reply:
Denne heter fortsatt catalogue i helm/gcp, @valosnah fikser det i en annen pr